packages/angular/build/src/builders/dev-server/vite-server.ts

```diff
@@ -854,7 +854,6 @@ export async function setupServer(
         outputFiles,
```

Comment:
Can you also please remove from `skipViteClient` from packages/angular/build/src/tools/vite/plugins/angular-memory-plugin.ts?

Reply:
adjusted the option name to better reflect its new behavior.